charts/k8s-monitoring/templates/_deprecations.tpl

```diff
@@ -11,7 +11,7 @@
 
 {{/* v0.9.0: allowLists replaced by metricsTuning */}}
 {{- if or
-  (index .Values.metrics.agent "allowList")
+  ((index .Values.metrics "agent").allowList)
```

Comment:
I'm not sure but just double checking: is `agent` correct here?

Reply:
Yeah, because this is checking for deprecated things from before. We used to have:
```yaml
metrics:
  agent:
    allowList: ...
 ```
 
 And now it's:
 ```yaml
metrics:
  agent:
    metricsTuning:
      useAllowList: ...
```

The deprecation warning still stands.

I'll make another deprecation warning that checks for old usage of `grafana-agent`, or `metrics.agent` in the values file.